### PR TITLE
credits: update links to the full list of code and docs contributors

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

- Add a link to tidb community page.
- Remove contributor lists.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: credits.md [hunk 1]
```diff
@@ -6,176 +6,6 @@ aliases: ['/docs-cn/dev/credits/']
...
-|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[Rustin-Liu](https://github.com/Rustin-Liu)|
-|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[spongedu](https://github.com/spongedu)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[Deardrops](https://github.com/Deardrops)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[foreyes](https://github.com/foreyes)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[lonng](https://github.com/lonng)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[SeaRise](https://github.com/SeaRise)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[tiancaiamao](https://github.com/tiancaiamao)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[wshwsh12](https://github.com/wshwsh12)|
+关于 TiDB 开发者社区 SIG 信息以及贡献者荣誉列表，请查阅 [SIG | TiDB DevGroup](https://developer.tidb.io/SIG/)。
```
Comment (yikeke) [inline credits.md:11]: 他们的页面还有什么介绍的话吗？可以多摘取些放到这里，现在这文档内容太少了。

Comment (ran-huang) [reply credits.md:11]: 他们页面上主要是介绍 SIG 的，我加一些对社区不同角色的介绍吧

Comment (ran-huang) [reply credits.md]: @yikeke @qiancai  PTAL~

**Comment by yikeke** [comment]
4.0 版本就不用改了，4.0 文档的描述没错

File: credits.md (hunk 1 above)
Comment (yikeke) [reply credits.md]: 突然觉得这个文档的性质不能叫做 credits.md 了

**Comment by ran-huang** [comment]
/sig docs sql-infra

test

File: credits.md [hunk 2]
```diff
@@ -6,176 +6,39 @@ aliases: ['/docs-cn/dev/credits/']
...
+- [pingcap/ticdc](https://github.com/pingcap/ticdc/graphs/contributors)
+- [pingcap/tidb-tools](https://github.com/pingcap/tidb-tools/graphs/contributors)
+- [pingcap/tidb-lightning](https://github.com/pingcap/tidb-lightning/graphs/contributors)
+- [pingcap/tispark](https://github.com/pingcap/tispark/graphs/contributors)
+- [pingcap/dumpling](https://github.com/pingcap/dumpling/graphs/contributors)
+- [tikv/client-java](https://github.com/tikv/client-java/graphs/contributors)
+- [tidb-incubator/TiBigData](https://github.com/tidb-incubator/TiBigData/graphs/contributors)
+- [ti-community-infra](https://github.com/orgs/ti-community-infra/people)
+
+完整的贡献者名单，请查阅 [SIG | TiDB DevGroup](https://developer.tidb.io/SIG/)
```
Comment (qiancai) [inline credits.md:34]:
```suggestion
完整的贡献者名单，请查阅 [SIG | TiDB DevGroup](https://developer.tidb.io/SIG)
```

**Comment by ran-huang** [comment]
@qiancai Any other comments?

/unhold

File: credits.md [hunk 3]
```diff
@@ -6,176 +6,39 @@ aliases: ['/docs-cn/dev/credits/']
...
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[foreyes](https://github.com/foreyes)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[lonng](https://github.com/lonng)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[SeaRise](https://github.com/SeaRise)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[tiancaiamao](https://github.com/tiancaiamao)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[wshwsh12](https://github.com/wshwsh12)|
+每一位贡献者都是推动 TiDB 健壮发展的重要成员，我们感谢所有为 TiDB 提交代码、撰写或翻译文档的贡献者。
+
+## TiDB 开发者
+
+TiDB 开发者为 TiDB 的新功能开发、性能优化、稳定性保障做出了贡献。以下是 TiDB 相关 repo 的贡献者名单：
```
Comment (qiancai) [inline credits.md:13]:
```suggestion
TiDB 开发者为 TiDB 的新功能开发、性能优化、稳定性保障做出了贡献。以下链接包含了 TiDB 相关 repo 的贡献者名单：
```

File: credits.md [hunk 4]
```diff
@@ -6,176 +6,39 @@ aliases: ['/docs-cn/dev/credits/']
...
+- [pingcap/dumpling](https://github.com/pingcap/dumpling/graphs/contributors)
+- [tikv/client-java](https://github.com/tikv/client-java/graphs/contributors)
+- [tidb-incubator/TiBigData](https://github.com/tidb-incubator/TiBigData/graphs/contributors)
+- [ti-community-infra](https://github.com/orgs/ti-community-infra/people)
+
+完整的贡献者名单，请查阅 [SIG | TiDB DevGroup](https://contributor.tidb.io/SIG)
+
+## TiDB 文档写作者和译员
+
+TiDB 文档写作者和译员为 TiDB 及相关项目撰写文档、提供翻译。以下是 TiDB 文档相关 repo 的贡献者名单：
```
Comment (qiancai) [inline credits.md:38]:
```suggestion
TiDB 文档写作者和译员为 TiDB 及相关项目撰写文档、提供翻译。以下链接包含了 TiDB 文档相关 repo 的贡献者名单：
```

**Comment by qiancai** [comment]
other LGTM

**Comment by ran-huang** [comment]
friendly ping @qiancai

**Comment by TomShawn** [comment]
@ran-huang @qiancai Can we merge this PR?

File: credits.md [hunk 5]
```diff
@@ -6,176 +6,39 @@ aliases: ['/docs-cn/dev/credits/']
...
+- [pingcap/ticdc](https://github.com/pingcap/ticdc/graphs/contributors)
+- [pingcap/tidb-tools](https://github.com/pingcap/tidb-tools/graphs/contributors)
+- [pingcap/tidb-lightning](https://github.com/pingcap/tidb-lightning/graphs/contributors)
+- [pingcap/tispark](https://github.com/pingcap/tispark/graphs/contributors)
+- [pingcap/dumpling](https://github.com/pingcap/dumpling/graphs/contributors)
+- [tikv/client-java](https://github.com/tikv/client-java/graphs/contributors)
+- [tidb-incubator/TiBigData](https://github.com/tidb-incubator/TiBigData/graphs/contributors)
+- [ti-community-infra](https://github.com/orgs/ti-community-infra/people)
+
+完整的贡献者名单，请查阅 [SIG | TiDB DevGroup](https://contributor.tidb.io/SIG)
```
Comment (qiancai) [inline credits.md:34]:
```suggestion
完整的贡献者名单，请查阅 [SIG | TiDB DevGroup](https://contributor.tidb.io/sig)
```

**Review by qiancai — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**